### DISCUSSION TITLE
Old photo still viewable after being replaced.

### DISCUSSION THREAD

**Discussion by tonnax** [pocketbase/pocketbase#4056] [Q&A] [answered]
Problem: I have a File type field where user can upload a Single file which are mostly image format.
Whenever I replace the old uploaded image with the new one from Dashboard or using Dart SDK, that old image url still accessible (I can still see the image). Is this some kind of cache mechanism or I'm missing something here? 

Expectation: I want that old image to be deleted after being replaced.

Disclaimer: I have connected my S3-compatible object service on my Pocketbase.
<img width="599" alt="image" src="https://github.com/pocketbase/pocketbase/assets/140943390/65077c0e-fdfe-4072-9e4f-bbf172c1d434">

**Comment by ganigeorgiev** [comment, marked as answer]
> that old image url still accessible (I can still see the image)

I'm not able to reproduce it locally.

I suspect that it is most likely a HTTP cache and you can safely ignore it.
If you are testing in the browser try refreshing with Ctrl+Shift+R.

Note that in some situations when using S3 the file may not be deleted immediately from the S3 bucket (some providers even have revision/delete policies), but in all cases when accessing the files via the `/api/files` PocketBase endpoint the deleted files shouldn't be accessible.

**Reply by tonnax** [reply]
<img width="624" alt="SCR-20231231-rtof" src="https://github.com/pocketbase/pocketbase/assets/140943390/7d5c327c-4d19-4a88-8247-0612d3fd3356">

Ah yes, after Ctrl+Shift+R and its gone. Thanks and happy new year Gani!
